documentation/APIbacklog.md

```diff
@@ -543,7 +562,19 @@ This is a live doc that captures the status of all the APIs which have been form
         <a href="https://github.com/camaraproject/DeviceDataVolume">DeviceDataVolume</a><br>
         <a href="https://github.com/camaraproject/DeviceDataVolume/blob/main/MAINTAINERS.MD">Maintainers</a>
       </td>
-      <td>No release</td>
+      <td>
+        <a href="https://github.com/camaraproject/DeviceDataVolume/releases/tag/r1.1">v0.1.0-alpha.1 (Pre-release) spring25</a><br>2025/01/16
+      </td>
+    </tr>
+    </tr>
+      <th>Best Interconnection</th>
+      <td>Deutsche Telekom*<div>Deutsche Telekom, GSMA-OGW</div></td>
+      <td>
+        <a href="https://github.com/camaraproject/APIBacklog/pull/88">Template</a><div>2024/01/19</div>
+      </td>
+      <td><ul><li>TSC Approved (2024/09/06)</li></ul></td>

```

Comment:
As far as I know "Best Interconnection" was never discussed nor approved within TSC and definitely not on in the [TSC 2024-09-05](https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14561510/2024-09-05+TSC+Minutes).  Please double-check also other approval statements - copy&paste errors?